packages/app-headless-cms/src/admin/components/ContentEntryForm/Header/DeleteEntry/useDeleteEntry.tsx

```diff
@@ -46,11 +46,11 @@ export const useDeleteEntry = (): UseDeleteEntryDialogResponse => {
                 });
 
                 if (error) {
-                    showDialog(error.message, { title: "Could not delete content!" });
+                    showDialog(error.message, { title: `Could not trash ${title}!` });
```

Comment:
`Couldn't move XYZ to trash!` ... could you please find a proper form online? Maybe Google Drive uses that word...